StackSplit/SS_stacksplit_start.m

```diff
@@ -65,19 +67,26 @@
 %
 % Roy et al. (2017), On the improvement of SKS splitting measurements by
```

Comment:
@michaelgrund do you like to keep the descending sorting by publication year or should we switch to an alphabetical sorting by authors?

Reply:
Keep it as is.

Reply:
OK!